lib/commands/vc.ts

```diff
@@ -0,0 +1,64 @@
+const ytdl = require("ytdl-core");
+export default async(home_scope: HomeScope){
+	const { message, args, CONFIG } = home_scope;
+	if(!message.guild) {
+		message.answer("Just use youtube-dl at home.");
+		return;
+	};
+	if(!CONFIG.vcqueue)
+		CONFIG.vcqueue = [];
+	switch(args[0]){
+		case "join":
+			if(message.member.voice.channel)
```

Comment:
This if statement has a body of two lines, but you didn not use braces (`{ ... }`).